purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1350,7 +1352,7 @@ class Purchases internal constructor(
                         ProductType.INAPP,
                         inAppProductIds,
                         { inAppProducts ->
-                            productsById.putAll(inAppProducts.map { it.productId to listOf(it) })
+                            productsById.putAll(inAppProducts.map { it.id to listOf(it) })
```

Comment:
minor: can we use the puschasingdata.productid like we do for subsicription products here? I know it's the same value, but i find it confusing that each uses a different id as th key for this map

Reply:
Yes yes, will make this update!